client/Router.js

```diff
@@ -26,6 +26,8 @@ import SwitchWithNotFound from './components/router/SwitchWithNotFound'
 
 const IsAdmin = requireRole([ADMIN_ROLE])
 const IsVolunteer = requireRole([ADMIN_ROLE, clientRoles.VOLUNTEER])
+const canInventory = requireRole([ADMIN_ROLE, volunteerRoles.INVENTORY])
+const canSchedule = requireRole([ADMIN_ROLE, volunteerRoles.SCHEDULE])
```

Comment:
There's a `volunteerRoles.PACKING` too

Reply:
Added a check for that as well. `IsVolunteer()` is no longer used, only more specific checks.

Reply:
Thanks!